kms-message/test/test_kmip_reader_writer.c

```diff
@@ -354,3 +355,156 @@ kms_kmip_reader_negative_int_test (void)
    kmip_reader_destroy (reader);
    free (data);
 }
+
+void
+kms_kmip_reader_find_test (void)
```

Comment:
Note: I think I should rename all of these test functions to remove the `kms_` prefix unless anyone objects. The new `kmip` types and functions do not have the `kms` prefix.